app/models.py

```diff
@@ -357,6 +357,49 @@ class Fido(Base, ModelMixin):
     __table_args__ = (sa.Index("ix_fido_user_id", "user_id"),)
 
 
+class AbuserData(Base, ModelMixin):
+    __tablename__ = "abuser_data"
+
+    user_id = sa.Column(sa.Integer, nullable=False, index=True)
+    encrypted_bundle = sa.Column(sa.LargeBinary(), nullable=False)
+
+    __table_args__ = (sa.Index("ix_abuser_data_id", "id"),)
+
+
+class AbuserLookup(Base, ModelMixin):
+    __tablename__ = "abuser_lookup"
+
+    hashed_address = sa.Column(sa.String(64), nullable=False, index=True)
+    abuser_data_id = sa.Column(
+        sa.Integer,
+        sa.ForeignKey("abuser_data.id", ondelete="cascade"),
+        nullable=False,
+        index=True,
+    )
+    bundle_k = sa.Column(sa.LargeBinary(), nullable=False)
+
+    __table_args__ = (
+        sa.Index(
+            "ix_abuser_lookup_hashed_address_abuser_data_id",
+            "hashed_address",
+            "abuser_data_id",
+        ),
```

Comment:
Shouldn't they be two independent indexes? Either we query by hashed address or by abuser_data_id, but afaics we won't query by both of them at the same time.